src/components/pages/itemDetail/itemDetail.jsx

```diff
@@ -0,0 +1,414 @@
+import defaultImage from '../../../assets/img_default.png'
+import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { getItemId, getProductComments } from '../../../api/Api';
+import { Link } from 'react-router-dom';
+
+/** @jsxImportSource @emotion/react */
+
+const pageWrapper = css`
+    max-width: 1200px;
+    margin: 0 auto;
+`
+
+const itemDetailWrapper = css`
+    margin: 32px 16px 42px;
+    gap: 24px;
+    display: flex;
+    padding-bottom: 16px;
+    border-bottom: 1px solid var(--gray-200);
+
+        @media(max-width: 425px) {
+            flex-direction: column;
+        }
+`
+const imgSection = css`
+    
+    > img {
+    width: 486px;
+    aspect-ratio: 1/1;
+    border-radius: 16px;
+    object-fit: cover;
+
+        @media(max-width: 768px) {
+            width: 340px;
+        }
+
+        @media(max-width: 425px) {
+            width: 100%;
+        }
+    }
+`
+
+const itemDetail = css`
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    gap: 24px;  
+`
+
+const itemHeader = css`
+    border-bottom: 1px solid var(--gray-200);
+    padding-bottom: 16px;
+    color: var(--gray-800);
+
+    
+    > h2 {
+        font-size: 40px;
+    }
+`
+
+const titleHeader = css`
+    font-size: 24px;
+    font-weight: 600;
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+`
+
+const editIconStyle = css`
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+`
+
+const itemDescription = css`
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    font-weight: 600;
+    color: var(--gray-600);
+    font-size: 16px;
+
+    > h3 {
+        font-weight: 600;
+        font-size: 16px;
+    }
+`
+
+const itemTag = css`
+    margin-bottom: 62px;
+    font-weight: 600;
+    color: var(--gray-600);
+
+    > h3 {
+        font-weight: 600;
+        font-size: 16px;
+    }
+
+    > span {
+        padding: 6px;
+    }
+`
+
+const tagWrapper = css`
+    display: flex;
+    flex-wrap: wrap;
+    gap: 8px;
+    margin-top: 16px;
+`
+
+const tagBox = css`
+    display: inline-flex;
+    padding: 6px 16px;
+    background: var(--gray-100);
+    border-radius: 26px;
+    border: none;
+
+    font-weight: 600;
+    color: var(--gray-600);
+
+    > span {
+        color: var(--gray-800);
+        font-weight: 400;
+    }
+
+    > h3 {
+        font-weight: 600;
+    }
+`
+
+const itemUserInfo= css`
+    display: flex;
+    align-items: center;
+    margin-top: 24px;
+`
+
+const userImg = css`
+    width: 40px;
+    height: 40px;
+    margin-right: 16px;
+
+    > img {
+        width: 40px;
+        height: 40px;
+    }
+`
+
+const userText = css`
+    font-weight: 500;
+    color: var(--gray-600);
+    font-size: 14px;
+    flex: 1;
+
+    > p {
+        margin-top: 6px;
+        font-weight: 400;
+        color: var(--gray-400);
+    }
+`
+
+const likeSection = css `
+    // display: inline-flex;
```

Comment:
❗️ 수정요청
사용하지 않는 코드는 지워주세요~

```suggestion
const likeSection = css `
```